tests/integration/test_api/conftest.py

```diff
@@ -83,8 +83,10 @@ def restart_api(get_configuration, request):
     file_monitor = FileMonitor(API_LOG_FILE_PATH)
     setattr(request.module, 'wazuh_log_monitor', file_monitor)
 
-    # Start Wazuh and Wazuh API
-    control_service('start')
+    # Start Wazuh API
+    for process_name in ['wazuh-apid', 'wazuh-modulesd', 'wazuh-analysisd', 'wazuh-execd', 'wazuh-db', 'wazuh-remoted']:
+        control_service('start', daemon=process_name)
```

Comment:
This looks to fail in centos8 vagrant environment, maybe it should be a good idea to start all wazuh.
However, this should be handled by the `daemon_handler` fixture. Update your branch with `4.2` and use it instead of this fixture.

In order to use it, you need to define the following variable in your module:

```
 daemons_handler_configuration = {'all_daemons': True}
```

And call to the fixture from in your test:
```
def test_max_upload_size(tags_to_apply, get_configuration, configure_api_environment, daemons_handler,
```

Also, the fixture `wait_for_start` should define its own FileMonitor as following:

```
file_monitor = FileMonitor(API_LOG_FILE_PATH)
```